example/spec/standalone/redeployment/TemplateOverrideDeploySpec.js

```diff
@@ -9,7 +9,7 @@ const {
 
 const { loadYmlFile } = require('../../helpers/configUtils');
 
-describe('When an iam override template is in the IAM directory ', () => {
+xdescribe('When an iam override template is in the IAM directory ', () => {
```

Comment:
Working on a separate PR for this. This broke because it's looking for `iam/config.yml` and we don't have that file anymore.

Reply:
Presumably the functionality isn't a concern, it's just the test setup that needs to be changed?

Reply:
yeah that is what i am thinking